dev/src/Command/AddComponentCommand.php

```diff
@@ -201,4 +242,73 @@ private function loadProtoContent(string $proto): string
         $response = $client->get($protoUrl);
         return (string) $response->getBody();
     }
+
+    private function bazelBuildLibrary(string $libraryName, string $googleApisDir): bool
+    {
+        $command = 'which bazel';
+        $output = exec($command, $output, $return);
+        if (strlen($output) == 0) {
+            exit("Error: Bazel is not installed. Please install Bazel before running this script.\n");
+        }
+        chdir($googleApisDir);
+        $command = "bazel query 'filter(\"-(php)$\", kind(\"rule\", //google/cloud/" . $libraryName .
```

Comment:
We should use the symfony "process" component for this. You can see an example of that happening in the [DocFxCommand](https://github.com/googleapis/google-cloud-php/blob/main/dev/src/Command/DocFxCommand.php#L147-L158)